cfgov/v1/models/story_page.py

```diff
@@ -0,0 +1,47 @@
+from wagtail.admin.edit_handlers import (
+    ObjectList, StreamFieldPanel, TabbedInterface
+)
+from wagtail.core.blocks import StreamBlock
+from wagtail.core.fields import StreamField
+from wagtail.core.models import PageManager
+
+from v1.atomic_elements import molecules, organisms
+from v1.models.base import CFGOVPage
+
+
+class StoryHeader(StreamBlock):
```

Comment:
Did you consider just importing `CampaignHeader` instead of redefining it here as `StoryHeader`? I may be missing something, but seems like that would be a little more DRY.

Reply:
I hear what you're saying but that feels risky because I can see story pages diverging from campaign pages in the future and some cross-contamination occurring. I suppose we could just redefine `StoryHeader` when that happens but it seems cleaner to keep them separate from the get-go.